app/client/src/sagas/__tests__/onPageUnloadSaga.test.ts

```
@@ -0,0 +1,394 @@
+                  {
+                    message: {
+                      name: "Error",
+                      message: "JS execution failed",
+                    },
+                    type: PLATFORM_ERROR.PLUGIN_EXECUTION,
+                  },
+                ],
+              },
+            },
+          ]);
+        });
```

Biome (1.9.4)

[error] This generator function doesn't contain yield.



(lint/correctness/useYield)